lib/commons/color/get-background-color.js

```diff
@@ -1,5 +1,5 @@
 /*global dom, color */
-/* jshint maxstatements: 31, maxcomplexity: 14 */
+/* jshint maxstatements: 32, maxcomplexity: 15 */
```

Comment:
I gave a shot at refactoring this to reduce complexity here: https://github.com/ckundo/axe-core/commit/2cac6de1dbcf2fdcbcebb2b3f636124cd6609619
I still had a failing test and gave up on it.